svm/src/account_loader.rs

```diff
@@ -270,18 +282,16 @@ fn load_transaction_accounts<CB: TransactionProcessingCallback>(
                             (default_account.data().len(), default_account, 0)
                         })
                 };
-                accumulate_and_check_loaded_account_data_size(
-                    &mut accumulated_accounts_data_size,
-                    account_size,
-                    tx_details.compute_budget_limits.loaded_accounts_bytes,
-                    error_metrics,
-                )?;
 
-                tx_rent += rent;
-                rent_debits.insert(key, rent, account.lamports());
-
-                account
-            };
+            accumulate_and_check_loaded_account_data_size(
+                &mut accumulated_accounts_data_size,
+                account_size,
+                tx_details.compute_budget_limits.loaded_accounts_bytes,
+                error_metrics,
+            )?;
+
+            tx_rent += rent;
+            rent_debits.insert(key, rent, account.lamports());
```

Comment:
to confirm, there's no effect of us now adding the instruction accounts rent debit here?

Reply:
Correct, since `rent == 0`, we won't actually insert anything. See `RentDebits::insert`:
```rust
    pub fn insert(&mut self, address: &Pubkey, rent_collected: u64, post_balance: u64) {
        if rent_collected != 0 {
            self.0.insert(
                *address,
                RentDebit {
                    rent_collected,
                    post_balance,
                },
            );
        }
    }
```